app-text/multitail/multitail-7.0.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic optfeature toolchain-funcs
+
+DESCRIPTION="Tail with multiple windows"
+HOMEPAGE="http://www.vanheusden.com/multitail/ https://github.com/folkertvanheusden/multitail/"
+SRC_URI="https://github.com/folkertvanheusden/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug examples unicode"
+RESTRICT="test" # bug 492270
+
+RDEPEND="sys-libs/ncurses:=[unicode(+)?]"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+  "${FILESDIR}"/${PN}-7.0.0-gentoo.patch
+  "${FILESDIR}"/${PN}-7.0.0-fix-clang16-build.patch
+  "${FILESDIR}"/${PN}-7.0.0-fix-lto-type-mismatch.patch
+)
+
+src_prepare() {
+	cmake_src_prepare
+	# cmake looks for licence.txt to install it, which does not exist in the package
+	cp LICENSE license.txt || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DUTF8_SUPPORT=$(usex unicode)
+	)
+	CMAKE_BUILD_TYPE=$(usex debug Debug)
```

Comment:
It's not a big deal for now but I'd either set this pre-inherit as it's an eclass var (preferred) or pass it into mycmakeargs with -D.

Reply:
How? I can't use the usex outside a function, can I? In the eclass man it says 
` If about to be set - needs to be set before invoking cmake_src_configure. ` so I guess that is why its down there?

Reply:
Oh, duh. Yes. :)